feature-utils/poly-look/test/react-components/cardList.test.js

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import { fireEvent, render } from "@testing-library/react";
+import { Card, CardList } from "../../src/react-components";
+import "@testing-library/jest-dom";
+import { HistoryStub } from "../utils/history-stub";
+import { INITIAL_HISTORY_STATE } from "../../src/constants/state";
+
+const testH1 = "TestH1";
+
+const singleBasicCard = (
+  <Card>
+    <h1>{testH1}</h1>
+  </Card>
+);
+
+//testing navigation
+let clickedCard;
+const history = new HistoryStub();
+const testRoute = "/testRoute";
+const testStateChange = { scrollingProgress: 200 };
+const testOnClick = () => {
+  clickedCard = true;
```

Comment:
Will clicking toggle on and off?

Reply:
No just wanted something thruthy to check whether it's clicked. Would you prefer "clicked" ?

Reply:
Ah, of  course. Clicking will do whatever we want. No, I'm good :-)